### ISSUE TITLE
About the example

### ISSUE THREAD

**Issue by hyper-carrot** [mailgun/godebug#17]
Hi,

I typed following code in the example dir:
```
 godebug -w .
```
And got:
```
godebug is a tool for debugging Go programs.

Usage:

        godebug command [arguments]

The commands are:

    run       compile, run, and debug a Go program
    test      compile, run, and debug Go package tests
    output    generate debug source code, but do not build or run it

Use "godebug help [command]" for more information about a command.
```

So the usage of example/README.md is incorrect.

**Comment by jeremyschlatter** [comment]
Good catch, thanks! I forgot to update that README when the command changed.

**Comment by jeremyschlatter** [comment]
Fixed in 980a46a7caf1a722cc2e520b1e394219b2376b8a

**Comment by hyper-carrot** [comment]
My pleasure.:)
